src/NServiceBus.Gateway/Settings/GatewaySettings.cs

```diff
@@ -36,5 +37,38 @@ public void ChannelFactories(Func<string, IChannelSender> senderFactory, Func<st
             config.GetSettings().Set("GatewayChannelSenderFactory", senderFactory);
             config.GetSettings().Set("GatewayChannelReceiverFactory", receiverFactory);
         }
+
+
+        /// <summary>
+        /// Set the number of retries and time increase between them for messages failing to be sent through the gateway.
+        /// </summary>
+        /// <param name="numberOfRetries">The total number of retries to do. 0 means no retry.</param>
```

Comment:
No . We pass current retry number into the custom policy and the `DefaultRetryPolicy` checks to see if that number is bigger this one. So if you call this with `1` (the very first retry) will be `if(1 > 1) no retry else 1 * timeIncrease`. I'll see if I can add some unit tests for the default retry policy.